src/Components/AlbumCardsDisplay/AlbumCardsDisplay.js

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from 'react'
+import './AlbumCardsDisplay.css'
+import AlbumCard from '../AlbumCard/AlbumCard';
+
+
+function AlbumCardsDisplay({ albums }) {
+
+ const [currentAlbums, setCurrentAlbums] = useState()
```

Comment:
Might be my stylistic choice but I think we should pass a default argument here, for example:
<code>useState([])</code>